Mopy/mash/mosh.py

```diff
@@ -1182,9 +1182,13 @@ def dumpData(self,out):
             for record in objRecords:
                 #--FRMR/NAME placeholder?
                 if isinstance(record, Cell_Frmr):
-                    out.pack('4si','FRMR',4)
-                    out.write(struct.pack('i',iObj)[:3])
-                    out.pack('B',iMod)
+                    if iMod > 255:
+                        out.pack('4si', 'FRMR', 8)
+                        out.write(struct.pack('2i', (iMod, iObj)))
+                    else:
+                        out.pack('4si', 'FRMR', 4)
+                        out.write(struct.pack('i', iObj)[:3])
+                        out.pack('B', iMod)
```

Comment:
Is there a reason this is inside the if/else statement? 
Shouldn't it be outside to get both situations?

==> out.pack('B', iMod) 

@NullCascade  @Greatness7

Reply:
In the `if iMod > 255` branch we write both `iMod` and `iObj` in the same line as they are both the same size (`2i` meaning it's just 2 integers). In the `else` branch they are written on separate lines as they are of different sizes. Note that the `Tes3Writer.pack` method is rather poorly named, it does both _packing and writing_, unlike `struct.pack` which does only the former.